descriptors/ModuleDescriptor-template.json

```diff
@@ -368,25 +381,24 @@
     {
       "permissionName": "user-settings.custom-fields.collection.put",
       "displayName": "Custom Fields - put collection",
-      "description": "Put Custom Fields collection",
-      "subPermissions" : [
-        "users.item.get"
-      ]
+      "description": "Put Custom Fields collection"
     },
     {
       "permissionName" : "users.item.get",
       "displayName" : "users item get",
       "description" : "Read an individual record in the User module"
     },
     {
-      "permissionName" : "users.read.basic",
+      "permissionName" : "users.basic-read.execute",
       "displayName" : "users read-basic",
-      "description" : "Read non-restricted User data information"
+      "description" : "Read non-restricted User data information",
+      "replaces" : ["users.read.basic"]
```

Comment:
Ensure that you update the permission names in all affected backend modules. This includes all permissions that have been modified. It would be beneficial to link all related PRs to this one in the TODOS or approach section of the PR.

Reply:
https://github.com/search?q=org%3Afolio-org%20users.read.basic&type=code

Reply:
I see its usage in mod-users, mod-scheduler and test spaces of other module as well and I believe they should be updated. Please check for all those permissions that have been renamed.

Reply:
> mod-users

I do changes in mod-users

> mod-scheduler

it's app descriptor that points to previous release version, not to snapshot, they'll need to do it by themself

> test spaces
I don't care about unit tests, as for postman test, nobody uses them, we use karate for long time